packages/insomnia/src/ui/components/rendered-query-string.tsx

```diff
@@ -136,11 +140,29 @@ export const RenderedQueryString: FC<Props> = ({ request }) => {
     }
   }, [tooLong]);
 
+  const showPreferneces = useCallback(() => {
```

Comment:
```suggestion
  const showPreferences = useCallback(() => {
```
it is preferable just to inline a state setter like this to the markup below, saves a bunch reading for the next guy for almost no perf benefit.